dtfs-central-api/src/helpers/calculate-fixed-fee-from-days-remaining.ts

```diff
@@ -0,0 +1,34 @@
+import Big from 'big.js';
+import { CalculateFixedFeeFromDaysRemainingParams } from '@ukef/dtfs2-common';
+
+/**
+ * An admin fee (fixed at 10%) is applied to the fixed fee, meaning
+ * we subtract this amount when calculating it
+ */
+const BANK_ADMIN_FEE_ADJUSTMENT = 0.9;
```

Comment:
Why does the documentation say 10% when the const is `0.9`?

Reply:
I am not sure, I moved this from an existing file